src/pages/home/sidebar/PressableAvatarWithIndicator.tsx

```diff
@@ -67,14 +48,13 @@ function PressableAvatarWithIndicator({isCreateMenuOpen, currentUserPersonalDeta
     );
 }
 
-PressableAvatarWithIndicator.propTypes = propTypes;
-PressableAvatarWithIndicator.defaultProps = defaultProps;
 PressableAvatarWithIndicator.displayName = 'PressableAvatarWithIndicator';
 export default compose(
```

Comment:
we should remove [compose](https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#compose-usage) during ts migration